packages/discount-code-generator/package.json

```diff
@@ -18,11 +18,21 @@
   "contributors": [
     "Williams Omayuku <williams.omayuku@commercetools.com>"
   ],
-  "main": "lib/index.js",
+  "main": "./lib/index.js",
```

Comment:
@hisabimbola That would be relative: In this repo, 4 packages are like this (this would be the 5th)

Also, check [this](#discussion_r114959894) out